packages/components/esbuild.config.js

```diff
@@ -1,7 +1,74 @@
 // @ts-nocheck
 /* eslint-disable import/no-extraneous-dependencies */
-const postCssPlugin = require("@deanc/esbuild-plugin-postcss")
-const esbuild = require("esbuild")
+import { existsSync, mkdirSync, readdirSync, statSync, writeFileSync } from "fs"
+import { join } from "path"
+import postCssPlugin from "@deanc/esbuild-plugin-postcss"
+import autoprefixer from "autoprefixer"
+import esbuild from "esbuild"
+import tailwindcss from "tailwindcss"
+import glob from "tiny-glob"
+
+const dist = join(process.cwd(), "dist")
+
+if (!existsSync(dist)) {
+  mkdirSync(dist)
+}
+
+// const entryPoints = readdirSync(join(process.cwd(), "src"))
+//   .filter(
+//     file =>
+//       file.endsWith(".ts") &&
+//       statSync(join(process.cwd(), "src", file)).isFile()
+//   )
+//   .map(file => `src/${file}`)
+
+// esm output bundles with code splitting
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+;(async () => {
```

Comment:
Starting with a `;`? 😛

Reply:
For real? 🥲 What if we remove both `;(` (wow, maybe the linter is telling us it's not happy)

Reply:
Seems it's some kind of JS optimisation.

> The leading `;`in front of immediately-invoked function expressions is there to prevent errors when appending the file during concatenation to a file containing an expression not properly terminated with a `;`.

https://stackoverflow.com/questions/1873983/what-does-the-leading-semicolon-in-javascript-libraries-do